apache-couchbase-mixin/README.md

```diff
@@ -52,6 +53,9 @@ scrape_configs:
           instance: "<your-instance-name>"
           cluster: "<your-cluster-name>"
           __path__: /opt/couchbase/var/lib/couchbase/logs/*.log
+    pipeline_stages:
+      - multiline:
+          firstline: '\[(ns_server|couchdb):(error|info),.*\]'
```

Comment:
Could it be  severities?  other than error or info? 
Same for ns_server|couchdb, could it be something else?

Reply:
@v-zhuravlev I don’t see anything in the docs to indicate any other prefixes or severities, but it's possible that there could be others under different circumstances. The Couchbase [logging docs](https://docs.couchbase.com/server/current/manage/manage-logging/manage-logging.html#log-file-listing) do not specify possible prefixes. These docs do specify that `couchdb.log` is the only file with a default log level of _info_. I am pretty confident in the log level/severity, but the possible prefixes are unknown outside of what I've observed in the raw log file in my environment